x-pack/plugins/alerting/server/lib/get_execution_log_aggregation.ts

```diff
@@ -102,6 +118,90 @@ const ExecutionLogSortFields: Record<string, string> = {
   num_new_alerts: 'ruleExecution>numNewAlerts',
 };
 
+export const getExecutionKPIAggregation = (filter?: IExecutionLogAggOptions['filter']) => {
+  const dslFilterQuery: estypes.QueryDslBoolQuery['filter'] = buildDslFilterQuery(filter);
+
+  return {
+    excludeExecuteStart: {
+      filter: {
+        bool: {
+          must_not: [
+            {
+              term: {
+                'event.action': 'execute-start',
+              },
+            },
+          ],
+        },
+      },
+      aggs: {
+        actionExecution: {
```

Comment:
As discussed with @XavierM, there was a bug related to filtering the execution outcome not counting the action outcomes correctly. Turns out we do need the bucket sort along with the 
```           
minExecutionUuidBucket: {
  bucket_selector: {
    buckets_path: {
      count: 'ruleExecution._count',
    },
    script: {
      source: 'params.count > 0',
    },
  },
},
```
snippet.